### PR TITLE
backport 4.3: now also supported on azure (#2722)

### PR DESCRIPTION
**Pull request by keichwa**
* payg now also supported on azure
  #2722 

# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
* Does this PR need to be backported? If yes, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed):

- master
- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Pay-as-you-go also on Azure supported
```
Comment (0rnela) [inline .changelog:7]:
```suggestion
- Added Pay-as-you-go as supported service on  Azure to the Public Cloud Guide
```

**Review by 0rnela — CHANGES_REQUESTED** [review]
Please change.

File: modules/specialized-guides/pages/public-cloud-guide/overview.adoc [hunk 2]
```diff
@@ -10,7 +10,8 @@ These procedures have been tested on {awslongform}, {azure}, and {gcplongform}.
 
 [NOTE]
 ====
-{payglongform} or {payg} services currently work solely on AWS. Solutions are in development for both {azure} and {gcplongform}.
+{payglongform} or {payg} services currently work on both {awslongform} and {azure}.
+Solutions are also in development for {gcplongform}.
```
Comment (0rnela) [inline modules/specialized-guides/pages/public-cloud-guide/overview.adoc:14]: Should we add this in the documentation? I don't think it's a good idea to "promise" something like this. The documentation is instructions, any other type of communication goes via other product channels.

**Review by 0rnela — CHANGES_REQUESTED** [review]
Please see comment.

File: modules/specialized-guides/pages/public-cloud-guide/overview.adoc (hunk 2 above)
Comment (keichwa) [reply modules/specialized-guides/pages/public-cloud-guide/overview.adoc:14]: To discuss it, and change it I created https://github.com/SUSE/spacewalk/issues/23547

**Pull request merged**